lighthouse/routes/common/cherrypicked_plates.py

```diff
@@ -41,12 +39,9 @@
 
 logger = logging.getLogger(__name__)
 
-bp = Blueprint("cherrypicked-plates", __name__)
-CORS(bp)
 
-
-@bp.get("/cherrypicked-plates/create")
-def create_plate_from_barcode() -> FlaskResponse:  # noqa: C901
+# TODO: Reduce the complexity of this method (C901) by splitting it up.
+def create_plate_from_barcode_v1() -> FlaskResponse:  # noqa: C901
```

Comment:
Same comment regarding `_v1`.

Reply:
👍 Please see the other comment for my reasoning.

Reply:
Cool, we can discuss there and resolve this one.